source-academy/cadet-frontend
# Additional fixes for GitHub Persistence

### Description
1. Added proper styling to the GitHub callback page
2. Changed the FileExplorerPanel to make use of 'setFilePath' rather than directly referring to the DOM element

### Type of change

### How to test
GitHub login now displays the proper styling during authentication and on failure. On successful login, the Callback page should terminate itself.

### Checklist
